src/fr/rappel.md

```diff
@@ -38,7 +38,7 @@ Le message de la fonction de rappel est formaté en JSON. Toutes les valeurs son
 |:---|:---|:---|
 |`id` | ID de GC Notification pour les accusés d’état  | UUID|
 |`reference` | Référence envoyée par le service | 12345678|
-|`to` | L’adresse de courriel ou numéro de téléphone du destinataire | hello@gov.uk ou 07700912345|
+|`to` | L’adresse de courriel ou numéro de téléphone du destinataire | hello@canada.ca ou 01234567890|
```

Comment:
It was only in the French version?

Reply:
Yeah, it's funny because the English version has been changed but not the French version: have a look at the English
https://github.com/cds-snc/notification-documentation/blob/main/src/en/callbacks.md